server/src/main/java/org/elasticsearch/index/IndexingSlowLog.java

```diff
@@ -76,6 +61,23 @@
     public static final Setting<SlowLogLevel> INDEX_INDEXING_SLOWLOG_LEVEL_SETTING =
         new Setting<>(INDEX_INDEXING_SLOWLOG_PREFIX +".level", SlowLogLevel.TRACE.name(), SlowLogLevel::parse, Property.Dynamic,
             Property.IndexScope);
+
+    private final Logger indexLogger;
+    private final Index index;
+
+    private boolean reformat;
+    private long indexWarnThreshold;
+    private long indexInfoThreshold;
+    private long indexDebugThreshold;
+    private long indexTraceThreshold;
+    /**
```

Comment:
A nit here, sorry: I know this is carried over from the code that you moved, but this is labeled as a Javadoc `/**` on a private field. It should be a regular comment `/*`. Also no reason to not make this comment use the full 140 columns that are available. 😇